SDKAndroid/src/main/java/app/cybrid/sdkandroid/core/BigDecimal.kt

```diff
@@ -43,10 +43,18 @@ class BigDecimal(internal val value:JavaBigDecimal) : Comparable<BigDecimal> {
         return BigDecimal(value.pow(n))
     }
 
+    fun toInt(): Int {
+        return value.toInt()
```

Comment:
There will be loss of precision here. We should raise an exception when that occurs (same with Double).

Reply:
Yeah!! But let me explain what happen here!

Our class BigDecimal its a child of java.math.BigDecimal, so the methos "toInt()" and "toDouble()" are methos from the parent and this methods call the exception for us, like the error that i show you to fix the OpenGenerator was from this class, for that reason i dont raise any exeption.

I added some test raising the exeption!